kotlintest-assertions/src/main/kotlin/io/kotlintest/matchers/FloatMatchers.kt

```diff
@@ -2,13 +2,35 @@ package io.kotlintest.matchers
 
 import io.kotlintest.Matcher
 import io.kotlintest.Result
+import io.kotlintest.shouldBe
+import io.kotlintest.shouldNotBe
 
 infix fun Float.plusOrMinus(tolerance: Float): FloatToleranceMatcher = FloatToleranceMatcher(this, tolerance)
 
 fun exactly(d: Float): Matcher<Float> = object : Matcher<Float> {
   override fun test(value: Float) = Result(value == d, "$value is not equal to expected value $d", "$value should not be equal to $d")
 }
 
+fun lt(x: Float) = beLessThan(x)
+fun beLessThan(x: Float) = object : Matcher<Float> {
+  override fun test(value: Float) = Result(value < x, "$value should be < $x", "$value should not be < $x")
+}
+
+fun lte(x: Float) = beLessThanOrEqualTo(x)
```

Comment:
These already exist in floats/matchers.kt.

The FloatMatchers.kt file is basically deprecated because it doesn't allow for multiple matchers of the same type but different type to be imported.

Reply:
Só, it's preferable to keep these rules in the floats package?

Reply:
Done!